src/umpire/interface/c_fortran/typesUmpire.h

```diff
@@ -1,9 +1,3 @@
-//////////////////////////////////////////////////////////////////////////////
-// Copyright (c) 2016-19, Lawrence Livermore National Security, LLC and Umpire
-// project contributors. See the COPYRIGHT file for details.
-//
-// SPDX-License-Identifier: (MIT)
-//////////////////////////////////////////////////////////////////////////////
```

Comment:
Did some tools change that caused this copyright to go missing?

Reply:
Yup, I can fix that.